src/HttpRequestParser.cpp

```diff
@@ -179,6 +192,13 @@ void HttpRequestParser::parseMethod(std::string &requestLine, HttpRequest *reque
 void HttpRequestParser::parseIndexPathAndDirectoryListing(HttpRequest *request, Server *server)
 {
 	std::string uri = request->getUri();
```

Comment:
no need to do now but you have the uri here but you still use request->getUri() on line 198..

Reply:
I think I did that on purpose

Reply:
wait the purpose was that I wanted to copy the functionality from line 225-227 and since it worked, i actually did not look more into it. but that can be changed as well